.github/workflows/code-check.yml

```diff
@@ -2,15 +2,22 @@ name: Code quality checks
 
 on:
   workflow_call:
+    inputs:
+      branch:
+        required: false
+        type: string
+        description: 'Branch to run on'
   push:
-    branches: [ main ]
+    branches: [ main, 0.x ]
 
 jobs:
   check-code:
     runs-on: ubuntu-latest
     steps:
     - name: Check out code
       uses: actions/checkout@v2
+      with:
```

Comment:
What will happen if action is triggered automatically from a push to 0.x branch? Will `inputs.branch` have a correct value?

Reply:
When empty it defaults to the "event" that triggered the run. In this case it will checkout 0.x branch.
https://github.com/actions/checkout#usage